ChangeLog.txt

```diff
@@ -209,6 +209,30 @@ MAP/HISTORY:
 - Added one historical governor of Mauretaniania Caesariensis
 - Added one historical governor of Lusitania
 - Added three historical governors of Tripolitania
+- Added five historical duces of Thebais
+- Added five historical duces et augustales of Alexandria
+- Added one historical Chalcedonian Patriarch of Alexandria
+- Fixed Valerius Faltonius Adelfius's wife name
+- Fixed Empress Aelia Eudocia having a wrong spouse added (Flavius Bauto)
+- Fixed age of Flavia Domitilla: she was born 9 years after her mother's death
+- Fixed age of Lucius Septimius Severus: he was born 15 years after his father's death
+- Fixed age of Aurelius Celsinus: he was born when his father was 5 years old
+- Fixed age of Flavius Probinus based on Geni
+- Fixed mother of Vibia Matidia Sabina based on Geni
+- Fixed Bonifacius being father of Sebastianus (his son-in-law)
+- Fixed Sidamic and Abasgian genealogies. Characters were born many years after their parents death
+- Fixed genealogy of Lakhmids (Al-Numan I not being son of Imru al-Qays II because they were born in same date)
+- Fixed genealogy of Quraishites (I know little about them, so its may be not perfect). Before this genealogy had many duplicate characters from vanilla, which i removed
```

Comment:
Do you need a family tree so we can double-check?

Reply:
Yes, i think you should double-check this.

Reply:
```suggestion
- Fixed genealogy of Quraishites (I know little about them, so it may be imperfect). Before this genealogy had many duplicate characters from vanilla, which I removed
```

Reply:
@IhateTrains That entry is not going to be kept like that anyway, the changelog is not in first person and we are bot going to say that one person knows little or not.

Reply:
I've edited the suggestion.

Reply:
I would further add, "it may still be imperfect because later parts are outside the time frame covered by history"

Reply:
I'm not sure if @Symmachus means that, though.